servers.json

```diff
@@ -73,5 +73,8 @@
   },
   {
     "address": "Chaotic-Neutral.ddns.net:3333"
+  },
+  {
+    "address": "Zap-hosting.com:36200"
```

Comment:
Same issue as before, I'm unable to connect to this.
![image](https://user-images.githubusercontent.com/45705145/89769739-07e49700-db41-11ea-8840-5d615c058f29.png)